### PR TITLE
Add random walk module

### PR DESCRIPTION
**Pull request by g-despot**
This module was created for an upcoming MAGE tutorial on [memgraph.com/docs](https://memgraph.com/docs). 

- [x] Add algorithm implementation.
- [x] Add `InsertPathValueResult` function to `mg_utils.hpp`.
- [x] Add e2e test.

### CODE REVIEW

File: cpp/random_walk_module/random_walk_module.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+/// @param result Memgraph result storage
+/// @param memory Memgraph memory storage
+void Generate(const mgp_list *args, const mgp_graph *graph, mgp_result *result, mgp_memory *memory) {
+  try {
+    auto start = mgp_value_get_vertex(mgp_list_at(args, 0));
+    auto length = mgp_value_get_int(mgp_list_at(args, 1));
+
+    auto path = mgp_path_make_with_start(start, memory);
+    auto vertex = start;
+    srand(time(NULL));
```
Comment (jmatak) [inline cpp/random_walk_module/random_walk_module.cpp:30]: I guess this shouldn't be in the release?

Comment (g-despot) [reply cpp/random_walk_module/random_walk_module.cpp:30]: Sorry, forgot about this PR for some time. 
I am probably missing something obvious so could you please elaborate? :smile:

Comment (jmatak) [reply cpp/random_walk_module/random_walk_module.cpp:30]: This module was made a while ago and would need an update to match the new API. The commented line has nothing to do with the question, sorry :sweat_smile:

Comment (g-despot) [reply cpp/random_walk_module/random_walk_module.cpp:30]: I get it now. :slightly_smiling_face: 
I will update it to the new API.

**Comment by jmatak** [comment]
Closing stale PR.

**Pull request closed without merging**